public/media-hunt/client.js

```diff
@@ -0,0 +1,8 @@
+const axios = require("axios");
+const instance = axios.create({
+  baseURL: "https://api.themoviedb.org/3/",
+  timeout: 2500,
+  params: {api_key: '109f8c8c06f943d5e337e7141b1dad67'}
+});
```

Comment:
This is incorrect. Please refer to postman or TMDB API documentation. Let's use a header, not param.

Reply:
am i supposed to use bearer token here?
example: curl --request GET \
  --url 'https://api.themoviedb.org/3/movie/76341' \
  --header 'Authorization: Bearer <<access_token>>' \
  --header 'Content-Type: application/json;charset=utf-8'

Reply:
header with token implemented, please review

Reply:
today i will deal with the secret token, therefore this userstory is still not completed

Reply:
NOTE: dotenv dependencie installed and env file placed in the root folder